.travis.yml

```diff
@@ -84,7 +84,7 @@ matrix:
     - os: linux
       env: CONFIG=python_compatibility
       sudo: required
-      dist: trusty
+      dist: xenial  # Required for Python 3.7
```

Comment:
Xenial is not natively available on Travis. You would need to use a Docker image.

See https://docs.travis-ci.com/user/reference/overview/

Reply:
At the bottom of travis-ci/travis-ci#9069 you will see that a substantial number of significant Python projects are already relying on Travis CI's Xenial virtual machines.  This PR recommends that we follow that trend.  If Travis CI removes or disables their existing Xenial and Bionic virtual machines then we could adopt the Docker image approach as a workaround.

Reply:
I missed that Xenial seems to be available now as some kind of public beta. Thanks for the link!